indexing-handlers/src/test/java/no/unit/nva/indexing/handlers/AllIndexHandlerTest.java

```diff
@@ -0,0 +1,55 @@
+package no.unit.nva.indexing.handlers;
+
+import static no.unit.nva.indexing.handlers.AllIndexHandler.SUCCESS_RESPONSE;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.StringContains.containsString;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import com.amazonaws.services.lambda.runtime.Context;
+import java.io.IOException;
+import no.unit.nva.search.IndexingClient;
+import no.unit.nva.search.IndicesClientWrapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+class AllIndexHandlerTest {
+
+    private AllIndexHandler allIndexHandler;
+    private IndexingClient indexingClient;
+    private Context context;
+
+    @BeforeEach
+    void init() {
+        indexingClient = mock(IndexingClient.class);
+        allIndexHandler = new AllIndexHandler(indexingClient);
+        context = mock(Context.class);
+    }
+
+    @Test
+    void shouldSentProperRequestToESToCreateIndexes() throws IOException {
```

Comment:
I am not sure you are testing what you are claiming here. How do we verify that the request is proper? The only thing we are testing is that the response we are getting from the handler is OK. 

If we are testing whether a request is proper, then we should get somehow this request and assert that its structure is the expected structure. We do not do this in this test. Here we are mocking the indicesClient and therefore we are basically accepting everything.  So what we are actually testing here is that the doNothing() method is actually not throwing an Exception.

I am not claiming that the test is wrong. I am not sure whether the name is in accordance with that is actually being tested. For me the test is more like "shouldNotThrowExceptionIfIndicesClientDoesNotThrowException".